.gitignore

```diff
@@ -10,4 +10,3 @@ project.vim
 .project
 .classpath
 .metadata/
```

Comment:
Those were in the base project, I didn't add them. But personally I prefer to keep the common ignores in the project specific ones, so it avoid having to deal with new developers that check in those by mistake because they didn't have a global ignore.